src/test/javascript/spec/component/shared/metis/postings-header/posting-header.component.spec.ts

```
@@ -0,0 +1,208 @@
+                MockComponent(ConfirmIconComponent),
+                MockComponent(ProfilePictureComponent),
+            ],
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(PostingHeaderComponent);
+        component = fixture.componentInstance;
+        debugElement = fixture.debugElement;
+        injector = fixture.debugElement.injector;
+    });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)